R/chunk.R

```diff
@@ -0,0 +1,23 @@
+#' Register Chunk Callback
+#'
+#' Registers a callback function to be executed after a specified chunk
```

Comment:
"after a specified chunk" seems to indicate that this function is bound only to specific chunks, but IIUC this will be run in response to any chunk that gets run. Is that correct?

If so, I think that could be clarified here ("to be executed after a chunk within an R Markdown document is run").